lib/stove-testing-e2e/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 dependencies {
     api(libs.kotlinx.core)
     api(libs.jackson.kotlin)
+    implementation(libs.jakson.datatype)
```

Comment:
We don't need this in implementation; you could take it to `testImplementation`.